src/components/calcite-loader/calcite-loader.tsx

```diff
@@ -20,9 +20,11 @@ export class CalciteLoader {
   //
   //--------------------------------------------------------------------------
   /** Show the loader */
-  @Prop({ reflect: true }) isActive: boolean = false;
+  @Prop({ reflect: true }) active: boolean = false;
   /** Inline loaders are smaller and will appear to the left of the text */
   @Prop({ reflect: true }) inline: boolean = false;
+  /** Speficy the scale of the loader. Defaults to "m" */
+  @Prop({ reflect: true }) scale: "s" | "m" | "l" = "m";
```

Comment:
@macandcheese @paulcpederson I think we had an example of a check somewhere that validated this prop and set it to the default if a rando value were passed yeah?